config/django/base.py

```diff
@@ -166,6 +168,8 @@
     'DEFAULT_AUTHENTICATION_CLASSES': []
 }
 
+SERVER_HOST_DOMAIN = env("SERVER_HOST_DOMAIN", default="http://localhost:8000")
```

Comment:
I need the host and the domain. Use url reverse?

Reply:
Even if using url reverse for serving the local upload url, when we need to serve the file.url, we need again  SERVER_HOST_DOMAIN. So dropping the idea for reverse

Reply:
https://stackoverflow.com/questions/2345708/how-can-i-get-the-full-absolute-url-with-domain-in-django